api-test/src/main/resources/pms/UpdateApiKeyExpiryDate/UpdateApiKeyExpiryDate.yml

```
@@ -0,0 +1,61 @@
+UpdateApiKeyExpiryDate:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)